content-warehouse/src/test/java/contentwarehouse/v1/DocumentTests.java

```diff
@@ -91,9 +91,9 @@ public void testUpdateDocument()
   @Test
   public void testSearchDocument()
       throws InterruptedException, ExecutionException, IOException, TimeoutException {
-    SearchDocuments.searchDocuments(PROJECT_ID, LOCATION, "document", USER_ID);
+    SearchDocuments.searchDocuments(PROJECT_ID, LOCATION, "warehouse", USER_ID);
     String got = bout.toString();
-    assertThat(got).contains("Document");
+    assertThat(got).contains("");;
```

Comment:
This testcase without assertion will throw only exceptions from the API.
Is that intended? Or is there any other way to make sure the sample is executed as expected?

Reply:
This is intended, every time a build runs for any sample, it creates a new document from another sample (CreateDocument). Since we were hitting a quota in the test project, I had to change the filter on the search. This test will search all documents for the text "warehouse". None of the documents have that text so it will return an empty string with no document names.

Reply:
Ah, got it. In that case, consider updating the statement to make an equals check for empty string.

Reply:
I just realized I could upload a document in the console. I changed this to search all docs for the string in this document so it's more relevant. I'll monitor the build to make sure the quota isn't reached